cmd/logcli/query.go

```diff
@@ -25,6 +25,14 @@ func doQuery() {
 
 	end := time.Now()
 	start := end.Add(-*since)
+	if *from != "" {
+		var err error
+		start, err = time.Parse(time.RFC3339, *from)
```

Comment:
For consistency, could this be changed to `time.RFC3339Nano`? That's the specific format `/api/prom/query` accepts and will still accept RFC-3339 strings without nanoseconds when parsing.